template/tests/behat/example.local.yml

```diff
@@ -54,15 +54,15 @@ local:
         # This must be an absolute path.
         drupal_root: ${docroot}
       drush:
-        alias: '@self'
+        alias: ${drush.aliases.local}
 
 example-multisite:
   extensions:
     Behat\MinkExtension:
       javascript_session: selenium2
       # Configure the base url for your site. For multisite, ensure that this
       # corresponds to a valid domain entry in sites.php.
-      base_url: http://127.0.0.1:8889
+      base_url: ${project.local.uri}
```

Comment:
This actually needs to be different than the local uri, since it's a multisite. It would needs it's own uri, hence the 8889 port rather than 8888.